resources/lang/vi/validation.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    "accepted" => "Trường :attribute phải được chấp nhận.",
+    "accepted_if"=> "Trường :attribute phải được chấp nhận khi :other là :value.",
```

Comment:
```suggestion
    "accepted_if" => "Trường :attribute phải được chấp nhận khi :other là :value.",
```
convention sửa lại tương tự những chỗ khác nhé